docs/deploying-kas-fleet-manager-to-openshift.md

```diff
@@ -147,6 +167,27 @@ make deploy/service IMAGE_TAG=<your-image-tag-here> <OPTIONAL_PARAMETERS>
 - `STRIMZI_OPERATOR_ADDON_ID`: The id of the Strimzi operator addon. Defaults to `managed-kafka-qe`.
 - `KAS_FLEETSHARD_ADDON_ID`: The id of the kas-fleetshard operator addon. Defaults to `kas-fleetshard-operator-qe`.
 
+### Using an Image from a Private External Registry
```

Comment:
I would like this section to be present before make deploy so that we can avoid deleting the pod. In a sense get the env ready and then make the deployment, WDYT?

Reply:
Unfortunately we can't because the service account we link the secret to is created with the template :(

Reply:
makes sense, for now lets go with this